Dockerfile.ubi

```diff
@@ -1,9 +1,12 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal
+FROM registry.access.redhat.com/ubi8/go-toolset AS builder
+
 ARG TARGETOS
 ARG TARGETARCH
 COPY bin/external-secrets-${TARGETOS}-${TARGETARCH} /bin/external-secrets
```

Comment:
This won't build the binaries within the container.

I just stumbled upon this article: https://kupczynski.info/posts/fips-golang/
turns out boringssl is now part of golang 1.19 which can be used through a `EXPERIMENT_` env var: https://github.com/golang/go/issues/51940.

Discussable if we want to do this :sweat_smile:

Reply:
This will be a way easier than rebuilding the build process.

Reply:
I opened #1731 to add a new FIPS compliant build, there were a couple of more changes needed.

Reply:
@moolen how far away are we from a new release that at least fixes the existing vulnerabilities?